packages/authenticator/amplify_authenticator/lib/src/mixins/authenticator_phone_field.dart

```diff
@@ -97,34 +98,37 @@ mixin AuthenticatorPhoneFieldMixin<FieldType extends Enum,
         suggestionsBuilder: ((context, SearchController controller) {
           final textStyle = Theme.of(context).listTileTheme.titleTextStyle ??
               const TextStyle(fontSize: 15);
-          final filteredCountries = countryCodes
+          final filteredCountries = DialCode.values
               .where(
                 (country) =>
                     country.value
                         .contains(controller.text.replaceFirst('+', '')) ||
-                    _countriesResolver
+                    _dialCodeResolver
                         .resolve(context, country.key)
                         .toLowerCase()
                         .contains(controller.text.toLowerCase()),
               )
-              .sortedBy(
-                (country) => _countriesResolver.resolve(context, country.key),
-              );
+              .toList()
+            ..sort(
```

Comment:
Any reason you chose this over `sortedBy`? They do the same, I think, but `sortedBy` is a lot cleaner and harder to mess up.